schemas/Action.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://raw.githubusercontent.com/valueflows/valueflows/master/json-schemas/Action.json",
+    "title": "Action",
+    "description": "An action verb defining the kind of event, commitment, or intent.",
```

Comment:
"Defining" seems more total while action is a particularity of events etc. Maybe
"defining what an event, commitment, or intent, does or commits or intends to do"?